Loop/View Controllers/InsulinDeliveryTableViewController.swift

```diff
@@ -2,13 +2,651 @@
 //  InsulinDeliveryTableViewController.swift
 //  Naterade
 //
-//  Created by Nathan Racklyeft on 3/11/16.
+//  Created by Nathan Racklyeft on 1/30/16.
```

Comment:
I think this file was moved to `LoopKit`, so this diff probably shouldn't be here.

Reply:
The original one was in LoopKit, but the new one is here; the old one hasn't been completely removed yet, but is named LegacyInsulinDeliveryTableViewController